lib/curly/presenter.rb

```diff
@@ -52,6 +52,10 @@ def initialize(context, options = {})
           end
         end
 
+        if value.respond_to?(:call)
+          value = value.call
```

Comment:
So yeah, we'd need to call the block once and store it in a hash or something. Could we use the proc as a key?